base_maintenance_group/security/maintenance_security.xml

```diff
@@ -21,4 +21,15 @@
         <field name="users" eval="[(4, ref('base.group_system'))]"/>
     </record>
 
+    <record model="ir.rule" id="only_subscribe_maintenance_team">
```

Comment:
Please place `id` attribute before `model`, as stated in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#xml-files:


```suggestion
    <record id="only_subscribe_maintenance_team" model="ir.rule">
```